src/components/titles/index.js

```diff
@@ -75,23 +75,58 @@ function draw(gd, titleClass, options) {
     var fontLineposition = font.lineposition;
     var fontShadow = font.shadow;
 
+    // Get subtitle properties
+    var subtitleProp = options.subtitlePropName;
+    var subtitleEnabled = !!(subtitleProp);
+    var subtitlePlaceholder = options.subtitlePlaceholder;
+    var subtitle = (cont.title || {}).subtitle;
+    var subtitleTxt = (subtitle && subtitle.text ? subtitle.text : '').trim();
+    var subtitleIsPlaceholder = false;
+    var subtitleOpacity = 1;
+
+    var subtitleFont = subtitle && subtitle.font ? subtitle.font : {};
+    var subFontFamily = subtitleFont.family;
+    var subFontSize = subtitleFont.size;
+    var subFontColor = subtitleFont.color;
+    var subFontWeight = subtitleFont.weight;
+    var subFontStyle = subtitleFont.style;
+    var subFontVariant = subtitleFont.variant;
+    var subFontTextcase = subtitleFont.textcase;
+    var subFontLineposition = subtitleFont.lineposition;
+    var subFontShadow = subtitleFont.shadow;
+
     // only make this title editable if we positively identify its property
     // as one that has editing enabled.
+    // Subtitle is editable if and only if title is editable
     var editAttr;
     if(prop === 'title.text') editAttr = 'titleText';
     else if(prop.indexOf('axis') !== -1) editAttr = 'axisTitleText';
     else if(prop.indexOf('colorbar' !== -1)) editAttr = 'colorbarTitleText';
     var editable = gd._context.edits[editAttr];
 
+    function matchesPlaceholder(text, placeholder) {
```

Comment:
You may declare this function above this scope. No?

Reply:
I could — would that be preferable? There are other helper functions defined within this scope so I was trying to copy the pattern of what's already there.

Reply:
There might be slight performance benefit. But OK let's keep them here to minimize the PR diff.
Resolving...